share/ansible_navigator/utils/catalog_collections.py

```diff
@@ -222,8 +216,8 @@ def process_directories(self) -> Tuple[Dict, List]:
         :returns: All collections found and any errors
         """
         for directory in self._directories:
```

Comment:
Should the class initializer accept an iterable of `Path` instead of strings for `self._directories`?